docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -7,22 +7,24 @@
 
 {description}
 
-NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage. See xref:deploy:enterprise-licenses.adoc[].
+NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage.
+See xref:deploy:enterprise-licenses.adoc[].
 
-== What Can You Do with Tiered Storage?
+Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API.
+It allows you to optimize infrastructure costs and performance by combining a fast, high performance in-memory layer and a lower-cost, high-capacity disk-based storage layer.
+With a larger data capacity per member, Tiered Storage can also reduce the number of deployed Hazelcast cluster members.
```

Comment:
```suggestion
With a larger data capacity per member, Tiered Storage can also reduce the number of deployed Hazelcast cluster members.
```
Tiered Storage allows you to work with datasets that are larger than memory using the same Hazelcast map API. It allows you to optimize infrastructure costs and performance by combining a fast in-memory layer with a cost-effective, high-capacity disk-based storage layer. 

Furthermore, with a extended data capacity per member, Tiered Storage can also reduce the number of deployed Hazelcast cluster members.

Reply:
Addressed in https://github.com/hazelcast/hz-docs/pull/1047/commits/85eb21450d0833a0e33da49a49d306678275a0ac.